src/config.js

```diff
@@ -20,14 +20,23 @@ const password = value => {
   }
 };
 
+convict.addFormat({
+  name: 'hex',
+  validate: function(val) {
+    if (/[^a-fA-F0-9]/.test(val)) {
+      throw new Error('must be a hex key');
+    }
+  }
+});
+
 const conf = convict({
   adminPass: {
     doc: 'The password for the admin user. Follow OWASP guidelines for passwords',
     format: password,
     default: 'invalid'
   },
   adminSessionSecret: {
```

Comment:
yeah, we discussed to do it in a separate patch because it's orthogonal and I didn't want to cram too much things into this patch. But I should probably revert the next change then.